src/components/shared/Text/AppText.tsx

```diff
@@ -1,11 +1,26 @@
 import React, { FC } from 'react';
+import { FontSize } from '../../../model/enums/FontSize';
+import './Text.scss';
 
 type AppTextProps = {
-  text: string
+  text: string,
+  color?: string,
+  fontSize?: FontSize,
 }
 export const AppText: FC<AppTextProps> = (props) => {
-  const { text } = props;
+  const { text, color, fontSize } = props;
+
+  let textClass = '';
+
+  if (fontSize) {
```

Comment:
could we use `classNames` instead of these conditions?

Reply:
Added classNames in commit d0df8953a6498f884fc8f239cf86721b7a4c4d9a